### ISSUE TITLE
Red Felix misunderstanding

### ISSUE THREAD

**Issue by WulfBloodfang** [ActualMandM/BRC-PostGameCharacters#2]
Checked release notes and saw that the most recent version removes Red Felix because of the most recent official game patch adding him post-game. Checking Steam forums has multiple people assuming the same thing and wondering where he is with an official response from the devs themselves saying that update only fixed him being unselectable if you swapped during the final stretch. He's not intended to be available afterwards, some people just bugged him in somehow and the devs don't even know how.

tl;dr devs say he isn't in post-game so re-enabling him in this mod would be good.

source: https://steamcommunity.com/app/1353230/discussions/1/3935643263197525975/

This is post-patch if you check dates.

**Comment by RagdollClash** [comment]
Interesting, I guess I'll need to make sure he doesn't get duplicated if he's already in the cypher then. Thanks for the heads up!

**Comment by RagdollClash** [comment]
Fixed with 1380f5980117685b82b8b874de1b2ba16aadbd82 (v1.0.4).
